### ISSUE TITLE
Error on terraform apply after stopping instance

### ISSUE THREAD

**Issue by ricardoguim** [terraform-google-modules/terraform-google-sql-db#286]
### TL;DR

After stopping the instance (changing the activation_policy = ‘NEVER’) the terraform script fails applying the script because it’s not able to read the SQL_USER data since the instance is stopped.

### Expected behavior

I see a couple options:

1. The module should be able to run (if there are no changes to the user) even if the instance is stopped, ignoring this resource (I don’t know a possible path to implement that since the google_sql_user resource depends on the instance to be running)
2. The module could have this `google_sql_user` as optional and it could be documented to not use it with the activation_policy = ‘NEVER’ 

### Observed behavior

When running terraform plan or terraform apply (after the instance is stopped) I see the following error:
Error: Error when reading or editing SQL User "username" in instance "SQL_DB_INSTANCE": googleapi: Error 400: Invalid request: Invalid request since instance is not running. 

### Terraform Configuration

```hcl
module "mssql" {
  source  = "GoogleCloudPlatform/sql-db/google//modules/mssql"
  version = "7.0.0"

  project_id                      = "dropfire"
  name                            = "sample-database"
  region                          = "us-central1"
  zone                            = "us-central1-a"
  ip_configuration = {
    authorized_networks = []
    ipv4_enabled        = true
    private_network     = null
    require_ssl         = null
  }
  deletion_protection             = false
  tier                            = "db-custom-2-8192"
  availability_type               = "REGIONAL"
  database_version                = "SQLSERVER_2019_STANDARD"
  disk_autoresize                 = true
  disk_size                       = 10
  user_name                       = "test_user"
  user_password                   = "*****"
  db_name                         = "test_db"
  db_collation                    = "SQL_Latin1_General_CP1_CI_AS"
  random_instance_name            = true
  activation_policy               = "NEVER"
  root_password                   = "*****"
}
```

### Terraform Version

```sh
Terraform v1.0.6
on darwin_arm64
+ provider registry.terraform.io/hashicorp/google v4.14.0
+ provider registry.terraform.io/hashicorp/google-beta v3.90.1
+ provider registry.terraform.io/hashicorp/null v3.1.1
+ provider registry.terraform.io/hashicorp/random v3.1.2
```
